docs/dev/adding_instruments.rst

```diff
@@ -510,6 +510,27 @@ Similar to `set_process` the :func:`Instrument.control <pymeasure.instruments.In
     >>> extreme.current
     3.1
 
+Another use-case of `set-process`, `get-process` is conversion from/to a :code:`pint.Quantity`. Modifying above example to set or return a quantity, we get:
```

Comment:
Awesome! I'd like to have a big ` .. note::` here emphasizing that this is a workaround for now -- I want to add proper support for properties like outlined in https://github.com/pymeasure/pymeasure/issues/248#issuecomment-1160009874 (i.e., have a `unit` parameter for our property factories, and some TBD features to control behaviour so we can transition nicely). I think we can either open a new issue on unit support for properties themselves, and point to that, or point to #248 (the former preferred).

Reply:
#666